File: ReparatieSysteem/ReparatieSysteem/Messaging/ShippingAddress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ReparatieSysteem.Messaging
{
    public class ShippingAddress
    {
        private String street;
        private Int32 number;
        private String postalCode;
        private String place;

        public String getStreet()
        {
            return street;
        }

        public Int32 getNumber()
        {
            return number;
        }

        public String getPostalCode()
        {
            return postalCode;
        }

        public String getPlace()
        {
            return place;
        }

        public ShippingAddress(String street, Int32 number, String postalCode, String place)
        {
            this.street = street;
            this.number = number;
            this.postalCode = postalCode;
            this.place = place;
        }

    }
}

File: ParafiksitDomain/src/main/java/com/marbl/client/domain/ShippingAddress.java
package com.marbl.client.domain;

import java.io.Serializable;

public class ShippingAddress  implements Serializable{

    private String street;
    private String number;
    private String postalCode;
    private String place;

    public ShippingAddress(String street, String number, String postalCode, String place) {
        this.street = street;
        this.number = number;
        this.postalCode = postalCode;
        this.place = place;
    }

    public String getStreet() {
        return street;
    }
    
    public void setStreet(String street) {
        this.street = street;
    }

    public String getNumber() {
        return number;
    }
    
    public void setNumber(String number) {
        this.number = number;
    }

    public String getPostalCode() {
        return postalCode;
    }
    
    public void setPostalCode(String postalCode) {
        this.postalCode = postalCode;
    }

    public String getPlace() {
        return place;
    }
    
    public void setPlace(String place) {
        this.place = place;
    }
}
